waterlog/src/reducers/PMapsReducer.js

```diff
@@ -1,39 +1,35 @@
-import {
-    FETCH_PMAP_DATA_BEGIN,
-    FETCH_PMAP_DATA_SUCCESS,
-    FETCH_PMAP_DATA_FAILURE
-} from '../actions/Types';
+import { FETCH_PMAP_DATA_BEGIN, FETCH_PMAP_DATA_SUCCESS, FETCH_PMAP_DATA_FAILURE } from '../actions/Types';
```

Comment:
This was correct, Please change it back to one import per line

Reply:
Fixed